build error: NoSuchMethodError: scala.Predef$.refArrayOps

when I run `mvn clean install` , I get this error-message:

`A NoSuchMethodError exception was thrown: scala.Predef$.refArrayOps`

I have not used scala before - I have installed sbt and the Scala plugin for IntelliJ.
Is there something else I need to install or some config that I am missing?

Detailed output:
```
[INFO] Vavr Parent ........................................ SUCCESS [  0.373 s]
[INFO] Vavr Match ......................................... SUCCESS [  1.976 s]
[INFO] Vavr ............................................... FAILURE [  8.922 s]
[INFO] Vavr Benchmark ..................................... SKIPPED
[INFO] Vavr Test .......................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
...
[ERROR] Failed to execute goal net.alchim31.maven:scala-maven-plugin:3.3.1:script (default) on project vavr: wrap: java.lang.Exception: A NoSuchMethodError exception was thrown: scala.Predef$.refArrayOps([Ljava/lang/Object;)[Ljava/lang/Object; -> [Help 1]
```

--- a/pom.xml
+++ b/pom.xml
@@ -83,8 +83,8 @@ We use these goals frequently to keep the dependencies and plugins up-to-date:
         <maven.exec.version>1.5.0</maven.exec.version>
         <maven.gwt.plugin>1.0-rc-6</maven.gwt.plugin>
         <scala.maven.version>3.3.1</scala.maven.version>
-        <scala.version>2.12.4</scala.version>
-        <scala.compat.version>2.12</scala.compat.version>
+        <scala.version>2.10.6</scala.version>
+        <scala.compat.version>2.10</scala.compat.version>
     </properties>
     <dependencyManagement>
         <dependencies>

--- a/pom.xml
+++ b/pom.xml
@@ -83,7 +83,8 @@ We use these goals frequently to keep the dependencies and plugins up-to-date:
         <maven.exec.version>1.5.0</maven.exec.version>
         <maven.gwt.plugin>1.0-rc-6</maven.gwt.plugin>
         <scala.maven.version>3.3.1</scala.maven.version>
-        <scala.version>2.12.2</scala.version>
+        <scala.version>2.12.4</scala.version>
+        <scala.compat.version>2.12</scala.compat.version>
     </properties>
     <dependencyManagement>
         <dependencies>
@@ -181,6 +182,7 @@ We use these goals frequently to keep the dependencies and plugins up-to-date:
                     <version>${scala.maven.version}</version>
                     <configuration>
                         <scalaVersion>${scala.version}</scalaVersion>
+                        <scalaCompatVersion>${scala.compat.version}</scalaCompatVersion>
                         <failOnMultipleScalaVersions>true</failOnMultipleScalaVersions>
                     </configuration>
                     <executions>

